bee-node/config.example.toml

```diff
@@ -47,8 +47,7 @@ status_interval = 10
 ms_sync_count = 200
 
 [rest_api]
-binding_port          = 14265
-binding_ip_addr       = "0.0.0.0"
+bind_addresses        = "/ip4/0.0.0.0/tcp/14265"
```

Comment:
why is it called `bind_addresses` and not `bind_address`? we only have a single binding address?

Reply:
You're not the only one confused 🙃

Reply:
I don't know ;) . I just applied the logic from @Alex6323. I think he made the same change somewhere else and I tried to be conform. The idea was, that with 0.0.0.0 you bind on all addresses. At least that's what I remember

Reply:
I understand the confusion, and I am not against changing it back to `bind_address` (singular) also for the network. It's worth mentioning though, that `0.0.0.0` in context of servers indeed means binding to potentially multiple addresses. For reference, read `https://en.wikipedia.org/wiki/0.0.0.0`.

Reply:
@Dr-Electron Can you also change it back to singular for the network then?

Reply:
I understand the reasoning, but even in your link they refer to this as a singular address.

Reply:
Sure. In principle you're right. It's an address, or meta-address, but it is not a valid bind-address in itself. In context of bind-addresses it usually describes a set of valid actual addresses. The reason I changed it in `bee-network` to plural was that if we follow Hornet's configuration, we should allow Bee to bind to multiple addresses, hence the plural. Currently we don't support that it a way that let's you pass an array of valid bind addresses. Instead we can bind to multiple addresses via the `0.0.0.0` meta-address. This is the section of Hornet's config:
```json
"p2p": {
    "bindMultiAddresses": [
      "/ip4/0.0.0.0/tcp/16600"
    ],
```

Reply:
Are we planning on actually supporting multiple addresses ? If yes, I'd already make it an array, even if not fully used.

Reply:
I think it makes sense to support that, yes. Not supporting it would probably even mean that we shouldn't allow `0.0.0.0`.

Reply:
Then let's do plural and array 👍

Reply:
okay, will add that to my todo list. @Dr-Electron I hope you didn't spend your time on this. We keep the plural in the network, and I will open a PR supporting multiple single multiaddresses. (lol)

Reply:
Ok, so I leave it as it is?

Reply:
yes, at least for the network leave it as it is.

Reply:
And for the API and dashboard? Because if we apply the same logic, it would also make sense there. I could theoretically have multiple network adapters and bind my dashboard to two of them, but not to all. I guess i will mark this as resolve than?